### PR TITLE
Save readonly mode of vector layers in project

### LINKED ISSUE #937: [Feature] Save readonly mode of vector layers in project
Original commit: https://github.com/qgis/QGIS/commit/1c59eff12dfe0aa8824cf0e89535864c48ef1041 by m-kuhn

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by yjacolin**
- Add a section on Identify configuration dialog
- Add information on read-only for fields via widget configuration

Fix #937

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -952,14 +952,20 @@ should be identified:
   identify features from. Operates like a right-click. Only the chosen features
   will be shown in the result panel.
 
+.. note:: **Identify configuration**
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: blank line

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -952,14 +952,20 @@ should be identified:
...
+   You can configure the identify feature in :menuselection:`Project -->
+   Project Properties` in the :guilabel:`Identify layers` tab. The table allow
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: allows

File: source/docs/user_manual/introduction/general_tools.rst [hunk 3]
```diff
@@ -952,14 +952,20 @@ should be identified:
...
+   user to select layer that can be used by this tools to find features
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: layers / tool

File: source/docs/user_manual/introduction/general_tools.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: blank line missing

File: source/docs/user_manual/introduction/general_tools.rst (hunk 3 above)
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: not sure `find features` is the most appropriate expression about this tool. No proposal, though

**Comment by SrNetoChan** [comment]
LGTM

**Comment by yjacolin** [comment]
It seems I have lost @DelazJ comments :(

**Comment by yjacolin** [comment]
I find the @DelazJ feedback apply the changes needed.

**Pull request merged**